command/pr.go

```diff
@@ -101,13 +101,19 @@ func prStatus(cmd *cobra.Command, args []string) error {
 	fmt.Fprintln(out, "")
 
 	printHeader(out, "Current branch")
-	if prPayload.CurrentPR != nil {
-		printPrs(out, 0, *prPayload.CurrentPR)
+	currentPR := prPayload.CurrentPR
+	currentBranch, _ := ctx.Branch()
+	noPRMessage := fmt.Sprintf("  There is no pull request associated with %s", utils.Cyan("["+currentPRHeadRef+"]"))
+	if currentPR != nil {
+		if baseRepo.(*api.Repository).DefaultBranchRef.Name == currentBranch && currentPR.State != "OPEN" {
```

Comment:
Note that this introduces a bug where gh will crash in case `baseRepo` is not an `api.Repository`, for example when `--repo` flag was used.

A value of type `ghrepo.Interface` promises only that it conforms to that interface, but does not guarantee any concrete type. Therefore, we can't write code that type-asserts a single type at runtime.